src/app/tab3/tab3.module.ts

```diff
@@ -1,26 +1,28 @@
 import { IonicModule } from '@ionic/angular';
 import { RouterModule } from '@angular/router';
-import { NgModule } from '@angular/core';
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { FormsModule } from '@angular/forms';
 import { Tab3Page } from './tab3.page';
 import { ExploreContainerComponentModule } from '../explore-container/explore-container.module';
+import { ReactiveFormsModule } from '@angular/forms';
 
 import { Tab3PageRoutingModule } from './tab3-routing.module';
 import { LoginComponent } from './login/login.component';
 import { RegisterComponent } from './register/register.component';
 import { ForgotPasswordComponent } from './forgot-password/forgot-password.component';
 import { ResetPasswordComponent } from './reset-password/reset-password.component';
+import { SharedModule } from '../shared/shared.module';
 
 @NgModule({
   imports: [
     IonicModule,
-    CommonModule,
-    FormsModule,
+    CommonModule,SharedModule,
+    FormsModule,ReactiveFormsModule,
```

Comment:
Add this Forms and Reactive Forms module in sharedModule and use by Shared Module

Reply:
Done